### DISCUSSION TITLE
Tab-navigator won't adapt new theme on scheme change

### DISCUSSION THREAD

**Discussion by Michota** [jpudysz/react-native-unistyles#226] [Q&A]
Hello. First of all: _świetna libka panowie_ ^^ (sorry english-only speakers :D). 
Second of all: how do I force Tabs to change colors along with other components of `expo-router` (which is based on `react-navigation`)?
It seems like every component inherits colors of new theme, but `<Tabs/>` does not, so I assume that's not Unistyles bug, but rather `react-navigation` one, but maybe there is a fix for that that some who already uses Unistyles knows how to fix?

https://github.com/jpudysz/react-native-unistyles/assets/109215822/34ba3bd6-5cce-431a-8e4d-6e9f37751d2b

**Comment by jpudysz** [comment]
Cześć Dawid 👋

Unistyles can't magically update views without styles passed as prop. Wherever you've used useStyle it worked as expected. You need to do the same with navigation components. 

The only issue is that navigation components don't accept style property. 

In this case you need to pass values a props like here: https://github.com/jpudysz/react-native-unistyles-expo-router/blob/main/app/_layout.tsx

I hope that will help to get you unblocked. 

Happy coding
